Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```
@@ -36,127 +36,217 @@
   - type: Sprite
     layers:
     - state: default
-    - state: idpassenger
+    - sprite: Interface/Misc/job_icons.rsi
```

This could also be aliased.
```suggestion
    - sprite: &icon-rsi Interface/Misc/job_icons.rsi
```
The layers would end up looking like this:
```
    - state: Captain
      sprite: *icon-rsi
      offset: *icon-offset
```